app/Hutch.Relay/Program.cs

```diff
@@ -1,6 +1,13 @@
+using Hutch.Relay.Data;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
+var connectionString = builder.Configuration.GetConnectionString("RelayDb");
```

Comment:
I realise now we don't have an `appsettings.development.json` to add the connection string to, could you add one? 
It should default to localhost here